docs/reference.rst

```diff
@@ -463,6 +463,10 @@ hierarchical separator.
   See table :ref:`sec-slot-type` for a more detailed list of these different types.
   Defaults to ``raw`` if none given.
 
+``extra-mkfs-opts=<options>`` (optional)
+  Allows to specify custom filesystem options that will be passed to the slot's
+  ``mkfs.type`` call (ext4, vfat, and ubifs only).
+
```

Comment:
Is there a reason to have it here instead of close to `extra-mount-opts`?

Reply:
I put it directly below `type` because you need to know the type to interpret the extra flags.